posthog/api/test/test_decide.py

```diff
@@ -3962,7 +3962,7 @@ def test_feature_flags_v3_consistent_flags(self, mock_is_connected):
         # make sure caches are populated
         response = self._post_decide()
 
-        with self.assertNumQueries(5, using="replica"), self.assertNumQueries(0, using="default"):
+        with self.assertNumQueries(9, using="replica"), self.assertNumQueries(0, using="default"):
```

Comment:
weird, how are there so many extra queries for flags, would you mind sharing the output here? I can look into this soon-ish as well

Reply:
oh its this

<img width="925" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/b2c41034-f00d-4a79-bb01-b8425bc70225">

I'll do a once-over anyway to confirm they're legit

Reply:
Yeah the increases should only be from this, the count changes do match that.